monday/query_joins.py

```diff
@@ -468,3 +468,81 @@ def get_complexity_query():
     }'''
 
     return query
+
+def get_complexity_query():
+    query = '''
+    query
```

Comment:
The appears to be an extra copy paste of the `get_complexity` method above - please remove.